.circleci/test-deploy.yml

```
@@ -77,6 +77,15 @@ jobs:
           name: "test command: install_sosh"
           command: VERSION="dev" INSTALL_DIR=~/bin ./test/commands/test_command_install_sosh.bats
 
+  test_macos_commands:
+    parameters:
+      executor: { type: executor }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)